### PR TITLE
 Adding initial MSSQL support for status dashboard  and Styling

### PR DESCRIPTION
**Pull request by Yasara123**

### CODE REVIEW

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/bean/WorkerResponce.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ */
+package org.wso2.carbon.status.dashboard.core.bean;
+
+/**
+ * Siddhi app Component Bean class.
+ */
+public class WorkerResponce {
+    private String message;
+    private int code;
+
```
Comment (gimantha) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/bean/WorkerResponce.java]: remove additional lines

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java [hunk 2]
```diff
@@ -347,7 +347,7 @@ public List selectAppOverallMetrics(String metricsType, String workerId, long
         switch (metricsType) {
```
Comment (gimantha) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java]: Better to move these to a query config file than hardcoding here.

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/impl/WorkersApiServiceImpl.java [hunk 3]
```diff
@@ -1105,7 +1106,8 @@ public Response getComponentHistory(String workerId, String appName, String comp
             StatusDashboardMetricsDBHandler metricsDBHandler = WorkersApi.getMetricStore();
             long timeInterval = period != null ? parsePeriod(period) : Constants.DEFAULT_TIME_INTERVAL_MILLIS;
             Map<String, List<List<Object>>> componentHistory = new HashMap<>();
-            if (timeInterval <= 3600000) {
```
Comment (gimantha) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/impl/WorkersApiServiceImpl.java]: 3600000 - move this as a constant

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/impl/WorkersApiServiceImpl.java [hunk 4]
```diff
@@ -1281,12 +1309,47 @@ public Response getWorkerConfig(String id, String username) throws NotFoundExcep
...
+            if (hostPort.length == 2) {
+                WorkerResponce workerResponce = new WorkerResponce();
+                String uri = generateURLHostPort(hostPort[0], hostPort[1]);
+                try {
+                    feign.Response workerResponse = WorkerServiceFactory.getWorkerHttpsClient(PROTOCOL + uri,
+                            getUsername(),
+                            getPassword()).getWorker();
+                    status = workerResponse.status();
+                    if (status == 200) {
+                        workerResponce.setCode(200);
```
Comment (gimantha) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/impl/WorkersApiServiceImpl.java]: spellcheck - reponce

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/bean/WorkerResponce.java (hunk 1 above)
Comment (Yasara123) [reply components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/bean/WorkerResponce.java]: Fixed in 84eed0f2066c8da5780a1ab6e5bbed953ff210b9

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java (hunk 2 above)
Comment (Yasara123) [reply components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java]: Fixed in 84eed0f2066c8da5780a1ab6e5bbed953ff210b9

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/impl/WorkersApiServiceImpl.java (hunk 3 above)
Comment (Yasara123) [reply components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/impl/WorkersApiServiceImpl.java]: Fixed in 84eed0f2066c8da5780a1ab6e5bbed953ff210b9

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/impl/WorkersApiServiceImpl.java (hunk 4 above)
Comment (Yasara123) [reply components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/impl/WorkersApiServiceImpl.java]: Fixed in 84eed0f2066c8da5780a1ab6e5bbed953ff210b9

**Pull request merged**